pkg/kvstore/kvstorepeerclient.go

```diff
@@ -66,6 +69,47 @@ func (proxy *KVPeerClient) RequestVote(req *raft.RequestVoteRequest, callback fu
 	}
 }
 
+// InstallSnapshot takes
+func (proxy *KVPeerClient) InstallSnapshot(req *raft.SnapshotRequest, callback func(*raft.AppendEntriesReply)) {
+	f, err := os.Open(req.File)
```

Comment:
if a client is down, there is no need to open the snapshot file. should we move this to be after getting the stream object?